tools/SourceKit/include/SourceKit/Core/LangSupport.h

```diff
@@ -138,6 +138,18 @@ struct ExpressionTypesInFile {
   StringRef TypeBuffer;
 };
 
+struct VariableType {
+  unsigned VarOffset;
+  unsigned VarLength;
+  unsigned TypeOffset;
```

Comment:
Since `TypeOffset` is not really self-explaining, could you add a doc-comment for it? Like
```cpp
/// The offset of the type’s string representation inside \c VariableTypesInFile.TypeBuffer
```